Cargo.toml

```diff
@@ -86,15 +86,39 @@ proptest-derive = { version = "0.7", default-features = false }
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive", "rc"] }
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
 serde-untagged = {version = "0.1" }
-sha2 = { version = "0.10", default-features = false }
 smallvec = { version = "1.15", default-features = false, features = ["union", "const_generics", "const_new"] }
 thiserror = { version = "2.0", default-features = false }
 tokio = { version = "1.48", default-features = false }
 tracing = { version = "0.1", default-features = false, features = ["attributes"] }
-winter-air = { version = "0.13", default-features = false }
-winter-fri = "0.13"
-winter-math = { version = "0.13", default-features = false }
-winter-prover = { version = "0.13", default-features = false }
-winter-rand-utils = "0.13"
-winter-utils = { version = "0.13", default-features = false }
-winter-verifier = { version = "0.13", default-features = false }
+rand = { version = "0.9", default-features = false }
+sha2 = { version = "0.10", default-features = false }
```

Comment:
Do we still need this dependency? We now should be able to use the wrappers from `miden-crypto` (though, this could be done in a separate PR).